ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ovn/ipsec/file/provision/cucushift-installer-rehearse-azure-ipi-ovn-ipsec-file-provision-chain.yaml

```diff
@@ -0,0 +1,14 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-ovn-ipsec-file-provision
+  steps:
+  - chain: ipi-conf-azure
+  - ref: ovn-conf
+  - ref: ovn-conf-ipsec-manifest
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
```

Comment:
seem like the same steps as `cucushift-installer-rehearse-azure-ipi-ovn-ipsec-provision`, so can we directly create the workflow yaml file to call `cucushift-installer-rehearse-azure-ipi-ovn-ipsec-provision` instead of adding this new provision chain file? just like what we did in `./gcp/ipi/filestore-csi/cucushift-installer-rehearse-gcp-ipi-filestore-csi-workflow.yaml`

Reply:
Sure, thanks I forgot update it.